modules/observe/adoption-insights/proc-viewing-active-users.adoc

```diff
@@ -0,0 +1,19 @@
+:_mod-docs-content-type: PROCEDURE
+[id="proc-viewing-active-users_{context}"]
+= Viewing the active users
+
+The *Active users* card displays the total number of active users over a specified time range. It also provides a breakdown comparison of *New users* and *Returning users* in a line/area graph format. You can export the user data in a .csv format.
+
+* *Returning users*: Existing users who have logged into {product-short}
+
+* *New users*: New users who have registered and logged into {product-short}
```

Comment:
Is this new users logging in for the first time or using different devices? More detail on new users might be helpful.